content/manuals/subscription/details.md

```diff
@@ -0,0 +1,345 @@
+- [Company layer](/admin/company/) to manage multiple organizations and settings
+- [Single sign-on](/security/for-admins/single-sign-on/)
+- [System for Cross-domain Identity
+  Management](/security/for-admins/provisioning/scim/)
+
+For a list of features available in each tier, see [Docker
+Pricing](https://www.docker.com/pricing/).
+
+### Self-serve
+
+A self-serve Docker Business subscription is where everything is set up by you.
+You can:
+
+- Manage your own invoices
+- Add or remove seats
+- Update billing and payment information
+- Downgrade your subscription at any time
+
+### Sales-assisted
+
+A sales-assisted Docker Business subscription where everything is set up and
+managed by a dedicated Docker account manager.
+
+{{< /tab >}}
+{{< tab name="Legacy subscriptions" >}}
+
+> [!IMPORTANT]
+>
+> As of November 15th, 2024, Docker Core, Docker Build Cloud, and Docker Scout
+> subscriptions are no longer available and have been replaced by consolidated
+> subscriptions that provide access to all tools. If you subscribed or renewed
+> your subscriptions before November 15th, 2024, your legacy Docker
+> subscriptions still apply to your account until you renew. For more details,
+> see [Announcing Upgraded Docker
+> Plans](https://www.docker.com/blog/november-2024-updated-plans-announcement/).
+
+
+The following describes some of the key features included with your legacy
+Docker Core, Docker Build Cloud, and Docker Scout subscriptions:
+
+![Docker Core subscription diagram](./images/subscription-diagram.webp)
+
+3 Docker Scout-enabled repositories for advanced remote image analysis. For more
+details, see [Docker Scout subscriptions](#legacy-docker-scout-subscriptions).
+
+Docker Build Cloud minutes are also included. For more details, see [Docker Build Cloud subscriptions](#legacy-docker-build-cloud-subscriptions).
+
+## Legacy Docker Core subscriptions
+
+### Docker Pro
+
+**Docker Pro** enables individual developers to get more control of their development environment and provides an integrated and reliable developer experience. It reduces the amount of time developers spend on mundane and repetitive tasks and empowers developers to spend more time creating value for their customers.
+
+Docker Pro includes:
+- Unlimited public repositories
+- Unlimited [Scoped Access Tokens](/security/for-developers/access-tokens/)
+- Unlimited [collaborators](/docker-hub/repos/access/#collaborators-and-their-role) for public repositories at no cost per month.
+- Access to [Docker Scout Free](#docker-scout-free) to get started with software supply chain security.
+- Unlimited private repositories
+- 5000 image [pulls per day](/docker-hub/download-rate-limit/)
+- [Auto Builds](/docker-hub/builds/) with 5 concurrent builds
+- 300 [Vulnerability Scans](/docker-hub/vulnerability-scanning/)
+
+For a list of features available in each tier, see [Docker Pricing](https://www.docker.com/pricing/).
```

Comment:
@craig-osterhout we are still waiting for the Marketing team to share the link to the legacy pricing page. Can we update the link text to indicate that it's the legacy pricing page?

Reply:
I started marking these as "TBD" in the commit below.